src/lib/Components/Bidding/Screens/ConfirmBid.tsx

```diff
@@ -172,7 +177,7 @@ export class ConfirmBid extends React.Component<ConfirmBidProps, ConfirmBidState
 
     commitMutation(this.props.relay.environment, {
       onCompleted: () => this.createBidderPosition(),
-      onError: errors => console.error("An error occurred in creditCardMutation: ", errors),
+      onError: this.presentErrorResult.bind(this),
```

Comment:
This error could also occur (imagining that we're only catching _certain_ errors from gravity in MP) for a variety of reasons, right? It seems a little fishy that we're always showing the "network" error. I realize these are the designs just noting since it could be confusing.

Reply:
Now that we have https://github.com/artsy/emission/pull/1118 in, the `onError` callback gets invoked only when the error type is an IO error. When Gravity returns an error to MP it will eventually be passed into the `onCompleted` callback as part of the `errors` argument. This is not handled right now, and is a case that will be covered by https://artsyproduct.atlassian.net/browse/PURCHASE-187.